test/thor_worker.cc

```diff
@@ -98,8 +98,7 @@ TEST(ThorWorker, test_linear_references) {
   std::vector<std::string> requests = {
       R"({"costing":"auto","linear_references":true,"locations":[
           {"lat":52.09110,"lon":5.09806},
-          {"lat":52.09098,"lon":5.09679}],
-          "action":"include"})",
+          {"lat":52.09098,"lon":5.09679}]})",
```

Comment:
that seemed redundant and actually crashed now that we do have an `action` parameter

Reply:
haha yeah im not sure how that got there. it was probably a copy past fail from a request with a `filter` in it which have actions of include and excluding filters

Reply:
should crash though, it should throw and then be an unhandled exception which crashes?

Reply:
not sure if this is a comment where you'd like to see smth changed or just a general notion :sweat_smile: but yeah, the exception will have code 115 and would be unhandled in this case